.ci/docker-compose-file/redis/certs/ca.txt

```diff
@@ -0,0 +1 @@
+9D5BD5E0B23D8FA4
```

Comment:
Is this being used somewhere?  I didn't find a mention of this elsewhere :see_no_evil:

Reply:
✅ I thought auxiliary files could be useful for smth., but the proved not to be, so we better remove them.

Reply:
Just curious: what this was?  Some kind of fingerprint for the CA certificate or something?

Reply:
This is a file for tracking certificates' serial numbers, it is created and used by these openssl options:
```
 -CAcreateserial - create serial number file if it does not exist
 -CAserial arg   - serial file
```